app/controllers/api/v2/remote_execution_features_controller.rb

```diff
@@ -28,6 +28,18 @@ def update
         process_response @remote_execution_feature.update_attributes(remote_execution_feature_params)
       end
 
+      def cockpit_redirect
```

Comment:
This should probably be in some other class.

Reply:
I'm preparing a patch to move it to standalone `CockpitController` so that we have all the pieces at one place. Will send it to you once ready.

Reply:
Please just push to my branch.

Reply:
Done